sdk/storage/azure-storage-blob/tests/test_block_blob.py

```diff
@@ -82,19 +83,41 @@ def _create_source_blob(self, data):
     def _get_bearer_token_string(self, resource: str = "https://storage.azure.com/.default") -> str:
         return "Bearer " + f"{self.get_credential(BlobServiceClient).get_token(resource).token}"
 
+    def _build_file_share_headers(self, bearer_token_string: str, content_length: int = 0) -> Dict[str, Any]:
```

Comment:
Can we make a small name change here to indicate this is a base set of headers since you sometimes modify it. Maybe just `_build_base_file_share_headers`